src/views/Recorder/EmptyState.tsx

```diff
@@ -60,23 +89,91 @@ export function EmptyState({ isLoading, onStart }: EmptyStateProps) {
           hint="Enter the URL of the website or service you want to test"
           hintType="text"
           errors={errors}
-          width="460px"
+          width="100%"
         >
-          <TextField.Root
-            {...register('url')}
-            placeholder="e.g. quickpizza.grafana.com"
-            autoFocus
-            css={css`
-              flex-grow: 1;
-              border-bottom-right-radius: 0;
-              border-top-right-radius: 0;
-            `}
-          />
+          <Flex>
+            <TextField.Root
+              {...register('url')}
+              placeholder="e.g. quickpizza.grafana.com"
+              autoFocus
+              css={css`
+                flex-grow: 1;
+                border-right: 0;
+                border-bottom-right-radius: 0;
+                border-top-right-radius: 0;
+              `}
+            />
+            <Button
+              disabled={isLoading || !canRecord}
+              type="submit"
+              css={css`
+                margin-left: -1px;
+                border-bottom-left-radius: 0;
+                border-top-left-radius: 0;
+              `}
+            >
+              {isLoading ? <Spinner /> : <DiscIcon />} Start recording
+            </Button>
+          </Flex>
         </FieldGroup>
-        <Button disabled={isLoading} type="submit">
-          <DiscIcon /> Start recording
-        </Button>
+        <WarningMessage
+          isProxyOnline={proxyStatus === 'online'}
```

Comment:
There's an interesting side-effect of this change: When the app starts and the proxy is not yet running, this warning will be displayed. I guess a better UX would be showing some like "Proxy is starting" in an info box - what do you think?

Reply:
@Llandy3d here's an idea:
1. Add another callback to `launchProxy`, something like `onStart`
2. Call it [here](https://github.com/grafana/k6-studio/blob/198ac6dd07fe21e073693ecdfcc77e0c6b07a410/src/proxy.ts#L72)
3. Change `restarting` to `starting`

Reply:
I would vote as well for showcasing the proxy status in an info box, similar on how we do on the sidebar on the lower left.
Might be enough to show the same value 🤔

Reply:
Here's the fix for the initial proxy status being `offline`: https://github.com/grafana/k6-studio/pull/524

Reply:
+1 for having the proxy status in an info box